qemu/riscv64.patch

```diff
@@ -59,16 +58,16 @@ Index: PKGBUILD
  
      _pick qemu-system-xtensa usr/bin/qemu-system-xtensa{,eb}
      _pick qemu-system-xtensa usr/share/man/man1/qemu-system-xtensa{,eb}.1*
-@@ -716,7 +722,7 @@
+@@ -711,7 +718,7 @@ package_qemu-hw-s390x-virtio-gpu-ccw() {
  
  package_qemu-system-aarch64() {
    pkgdesc="QEMU system emulator for AARCH64"
--  depends=("${_qemu_system_deps[@]}" edk2-armvirt systemd-libs libudev.so)
+-  depends=("${_qemu_system_deps[@]}" edk2-aarch64 systemd-libs libudev.so)
 +  depends=("${_qemu_system_deps[@]}" systemd-libs libudev.so)
```

Comment:
The `edk2-arm` dependency of `qemu-system-arm` package needs to be removed as well, or the package would become uninstallable.

Reply:
Sure, thanks for the input.